frontends/relay/relay_visitor.py

```diff
@@ -528,7 +533,12 @@ def get_program_dat_memories(relay_ir):
         Import("primitives/binary_operators.futil"),
         Import("primitives/math.futil"),
     ]
-    (dahlia_defs, prog) = emit_calyx(relay_ir)
+    if args.save_mem == "false":
```

Comment:
Nit: save_mem = args.save_mem == "true"

It might also be a good idea to lowercase it, so that `True` works here too.

Similar for frontends/relay/onnx_to_calyx.py